tests/test_converter.py

```diff
@@ -10,7 +10,6 @@
 from pennylane_forest.converter import *
 
 
-@pytest.mark.xfail
```

Comment:
The majority of the converter tests pass, with the exception of 6, each of which fail with:

```
AttributeError: 'QNode' object has no attribute 'jacobian'
```

One example of this is on line 1233:

```
assert np.array_equal(qml.jacobian([angle]), circuit_reg.jacobian([angle]))
```

This seems like a change on the PennyLane side, any idea on what we need to do to fix?

Reply:
Yes!

You need to use:
```python
assert np.array_equal(qml.jacobian(circuit)(angle), qml.jacobian(circuit_reg)(angle))
```

Instead of:
```python
assert np.array_equal(qml.jacobian([angle]), circuit_reg.jacobian([angle]))
```

Reply:
That did the trick, thanks!